serverless.yml

```diff
@@ -37,9 +37,9 @@ provider:
         - https://data.prod.umccr.org
         - https://data.dev.umccr.org
         - https://data.stg.umccr.org
-        - https://status.data.umccr.org
-        - https://status.data.prod.umccr.org
-        - https://status.data.dev.umccr.org
+        - https://status.umccr.org
+        - https://status.prod.umccr.org
+        - https://status.dev.umccr.org
```

Comment:
Could you also add `https://status.stg.umccr.org` pls.

Reply:
sure, added